src/components/Footer/Footer.module.css

```diff
@@ -0,0 +1,11 @@
+.footer {
+   height: 20vh;
+   font-family: Roboto;
+   color: white;
+   text-align: center;
+   display: flex;
+   flex-direction: column;
+   justify-content: center;
+   border-top: 1px solid #868e96;
+   background-color: #ced4da;
+}
```

Comment:
couldn't you use any of the multiple variables to do it?

also, is there a point on giving you a review about these kind of things or you would tell me that it would change in the future so it doesn't matter?
If it doesn't matter then it shouldn't be here (you could have clean it up first).